data/json/npcs/exodii/exodii_workers.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "type": "talk_topic",
+    "id": "TALK_EXODII_WORKER",
+    "dynamic_line": [
+      "&You see a human-like robot walking nearby.",
+      "&You see a cyborg moving somewhere in a hurry.",
```

```suggestion
      "&You see a mechanical humanoid moving somewhere in a hurry.",
```
Stuff like this. Other terms might be "humanoid machine" and stuff.